public/index.html

```diff
@@ -25,12 +23,24 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
-  </head>
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-    <!--
+  <!-- Favicon images -->
+  <link rel="icon" href="https://github.com/setlife-network/website/files/3630402/favicon_io.zip">
+
+  <!-- title & metadata for SEO -->
+  <title>SetLife | People over Profit</title>
+  <meta name="title" property="og:title" content="SetLife | People over Profit" />
+  <meta name="image" property="og:image"
+    content="https://user-images.githubusercontent.com/4914611/65231213-81aa0700-dacf-11e9-9450-1fdad1f71b76.png" />
+  <meta name="description" property="og:description"
```

Comment:
Made a last minute change to this copy if you could swap it out please:

“Learn valuable technical skills to upgrade your career or simply keep up with advancing technology”

Reply:
Ok, changed it.